src/screens/dashboard/components/Notification.tsx

```diff
@@ -22,43 +24,66 @@ export const Notification = ({
   const { mutate } = useMarkNotificationAsSeen()
   const opacity = wasSeenByHolder ? 0.6 : 1
 
+  // Comment: handle user timeOff type
+  const { allUsers } = useTeamsContext()
+  const [isModalVisible, setIsModalVisible] = useState(false)
+  const [modalUser, setModalUser] = useState<User>()
+
+  const openModal = () => {
+    const userModalData = allUsers.find(
+      (userData) => userData.firstName === author.firstName && userData.lastName === author.lastName
```

Comment:
Can we check it by id 🤔 ?

Reply:
In this case no as there is no common source for mocked userData and mocked author. I've added comment with to do when BE would be provided.